ui/admin/app/styles/app.scss

```diff
@@ -249,7 +249,7 @@
     margin-bottom: sizing.rems(l);
   }
 
-  > p code {
+  > p span {
     background-color: var(--token-color-surface-strong);
     border-radius: 5px;
     color: var(--token-color-foreground-primary);
```

Comment:
small nit: this color property can be passed into the component as `@color='primary'` instead.

Reply:
Oh yes. Updated!